docs/authentication/social-connections/apple.mdx

```diff
@@ -105,11 +117,30 @@ In _production instances_, you must provide custom credentials, which includes g
 
   After this, you should now see the email address added to the list, and it should be marked as verified with a green check icon. If it does not appear as verified yet, DNS propagation may take some time to complete so wait before trying to select the **Reverify SPF** button.
 
+  The **Email Source** and **Apple Private Relay** are required only for web based OAuth flows.
```

Comment:
Is this whole section about configuring the email source for web-based oauth flows only?

Reply:
Correct, the private relay/email configuration is only applicable to web based "traditional" OAuth flows.

Reply:
cool ! check out this commit and lmk what you think: [68a1a6a](https://github.com/clerk/clerk-docs/pull/1294/commits/68a1a6a60e39ec924cce69b372cea4c70af15cbb)

Reply:
Looks good to me!

Perhaps we could also mentioned it's "required" or do you think it's redundant?

```
This is an optional step and is required for web based OAuth flows only.
```

Anyway, looks good! :smile:

Reply:
oh its a required step ?? I thought it was optional. If it's required, then yes we should change it to be "This step is required for web-based OAuth flows only"